source/code/scxsystemlib/common/scxostypeinfo.cpp

```diff
@@ -52,7 +52,7 @@ using namespace SCXCoreLib;
 
 namespace {
 
-#if defined(linux)
+#if defined(linux) && defined(PF_DISTRO_SUSE)
```

Comment:
I'm not sure that I have access to pbuild at the moment.

Upon further examination, ExtractOSName() is used when 'linux' and PF_DISTRO_SUSE or PF_DISTRO_REDHAT are defined.  So my change should actually be:
#if defined(linux) && !defined(PF_DISTRO_ULINUX)